pkg/image/apis/image/v1/types.go

```diff
@@ -195,7 +197,8 @@ type TagReference struct {
 	Name string `json:"name" protobuf:"bytes,1,opt,name=name"`
 	// Annotations associated with images using this tag
 	Annotations map[string]string `json:"annotations" protobuf:"bytes,2,rep,name=annotations"`
-	// From is a reference to an image stream tag or image stream this tag should track
+	// Optional; if specified, a reference to another image that this tag should point to. Valid values
```

Comment:
Question - should the docstrings for other fields match ../types.go too?

Reply:
probably, will update.